ghcide/test/exe/Main.hs

```diff
@@ -1242,6 +1248,7 @@ pluginSimpleTests =
 
   -- Error: cabal: Failed to build ghc-typelits-natnormalise-0.7.7 (which is
   -- required by plugin-1.0.0). See the build log above for details.
+  ignoreFor (BrokenForGHC [GHC96]) "fragile, frequently times out" $
```

Comment:
issue?

Reply:
I think its fragile for all ghc versions, I think due to #3423